src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -149,90 +149,61 @@ them, and how to use them to configure your environment.
     point your browser to the _Your Web Services Account_ button, a drop
     down list displays. Click _View Access Key Identifiers_.
 
-4.  Use this information to fill in the properties *AWS_AKEY* (Access
-    Key), *AWS_SKEY* (Secret Key) and *AWS_USER* (numerical EC2 user
-    ID) in the file located in */config/rm/deployment/ec2.properties*.
-    Those thee parameters should never change, except if you need for
+4.  Use this information to fill in the properties *aws_key* (Access
+    Key), *aws_secret_key* (Secret Key) in the *Create Node Source* panel located in the resource manager.
+    Those two parameters should never change, except if you need for
     some reason to handle multiple EC2 accounts. Other properties in the
-    configuration file are:
-
-    -   *AMI:* Defines which AMI will be deployed on instances. The
-        value to provide is the unique AMI Id, ami-XXX. If no default
-        value is provided, you need to find a public ProActive AMI
-        corresponding to the release you are using.
-
-    -   *INSTANCE_TYPE:* One of m1.small, m1.large, m1.xlarge,
-        c1.medium, c1.xlarge, which defines what kind of hardware the
-        instance will be running on. m1 instances are memory focused,
-        whereas c1 instances are for more CPU intensive tasks. This
-        property defaults is m1.small, the cheapest and slowest of all.
+    *Create Node Source* are:
+    
+    -   *rmDomain:* The public IP address of the resource manager. This IP needs to be accessible from the AWS cloud. 
+    
+    -   *connectorIaasURL:* Connector-iaas is a service embedded in the scheduler used to interact with IaaS like AWS. By default it runs on the same domain and port of the scheduler.    
+
+    -   *image:* Defines which AMI will be deployed on the AWS instances to create. The
+        value to provide is the AWS region together with the unique AMI Id, for example : eu-west-1/ami-bff32ccc. 
```

Comment:
I cannot find a common example syntax in the existing doc. Could you point me to a correct one please? Thanks